Game/Source/ThirdPersonShooter/BigGym/BigGymGameMode.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "BigGymGameMode.h"
+#include "UObject/ConstructorHelpers.h"
+
+#include "EngineClasses/SpatialNetConnection.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Engine/World.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+
+ABigGymGameMode::ABigGymGameMode()
+{
+	PrimaryActorTick.bCanEverTick = true;
+
+	//set default pawn class to our Blueprinted character
+	static ConstructorHelpers::FClassFinder<APawn> PlayerPawnBPClass(TEXT("/Game/BigGym/EntityBlueprints/BigGymCharacter_BP"));
+	if (PlayerPawnBPClass.Class != NULL)
+	{
+		DefaultPawnClass = PlayerPawnBPClass.Class;
+	}
+	
+	// Seamless Travel is not currently supported in SpatialOS [UNR-897]
+	bUseSeamlessTravel = false;
+}
+
+//void ABigGymGameMode::Tick(float DeltaSeconds)
```

Comment:
You can remove this. Was from an experiment I was doing before Josh's actor replication throttling change went in